### PR TITLE
Add local ironic container re-create instructions

### PR DESCRIPTION
**Pull request by kashifest**
This PR adds how to recreate local ironic containers since the process is not trivial.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,4 +143,29 @@ Afterwards, you can deploy a target cluster.
...
+source lib/common.sh
+source lib/network.sh
+
+export IRONIC_HOST="${CLUSTER_URL_HOST}"
+export IRONIC_HOST_IP="${CLUSTER_PROVISIONING_IP}"
+
+source lib/ironic_tls_setup.sh
+source lib/ironic_basic_auth.sh
+
+cd$ ${BMOPATH}
```
Comment (furkatgofurov7) [inline README.md:163]:
```suggestion
cd ${BMOPATH}
```

**Review by furkatgofurov7 — COMMENTED** [review]
typo?

File: README.md (hunk 1 above)
Comment (kashifest) [reply README.md]: thanks. Done

**Review by maelk — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -143,4 +143,29 @@ Afterwards, you can deploy a target cluster.
 If you are running tilt on a remote machine, you can forward the web interface
 by adding the following parameter to the ssh command `-L 10350:127.0.0.1:10350`
 
-Then you can access the Tilt dashboard locally [here](http://127.0.0.1:10350)
\ No newline at end of file
+Then you can access the Tilt dashboard locally [here](http://127.0.0.1:10350)
+
+### Recreating local ironic containers
+
+In case, you want recreate the local ironic containers enabled with TLS, you
```
Comment (furkatgofurov7) [inline README.md:150]:
```suggestion
In case, you want recreate the local ironic containers enabled with TLS and Basic Auth, you
```

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]
Question inline

File: README.md (hunk 2 above)
Comment (kashifest) [reply README.md:150]: Thanks. Will take the change in another PR which I am doing. Don't want to waster two more CI runs on this.

**Pull request merged**